JSQMessagesViewController/Model/JSQVideoMediaItem.m

```diff
@@ -81,16 +88,13 @@ - (UIView *)mediaView
     }
     
     if (self.cachedVideoImageView == nil) {
-        CGSize size = [self mediaViewDisplaySize];
-        UIImage *playIcon = [[UIImage jsq_defaultPlayImage] jsq_imageMaskedWithColor:[UIColor lightGrayColor]];
         
-        UIImageView *imageView = [[UIImageView alloc] initWithImage:playIcon];
-        imageView.backgroundColor = [UIColor blackColor];
-        imageView.frame = CGRectMake(0.0f, 0.0f, size.width, size.height);
-        imageView.contentMode = UIViewContentModeCenter;
-        imageView.clipsToBounds = YES;
-        [JSQMessagesMediaViewBubbleImageMasker applyBubbleImageMaskToMediaView:imageView isOutgoing:self.appliesMediaViewMaskAsOutgoing];
-        self.cachedVideoImageView = imageView;
```

Comment:
let's keep this how it was, but allow clients to set an `image` to use.

then, in this class, let's add the `playIcon` on top. (so there would be 2 image views. 1 for the play icon, the other for a custom thumbnail)